libcaf_net/caf/net/multiplexer.test.cpp

```diff
@@ -257,30 +290,32 @@ SCENARIO("a multiplexer terminates its thread after shutting down") {
 //       alice->register_reading();
 //       bob->register_reading();
 //       apply_updates();
-//       CHECK_EQ(mpx->num_socket_managers(), 3u);
+//       check_eq(mpx->num_socket_managers(), 3u);
 //       THEN("the multiplexer swaps out the socket managers for the socket") {
 //         alice->send("Hello Bob!");
 //         alice->register_writing();
 //         exhaust();
-//         CHECK_EQ(bob->receive(), "Hello Bob!");
+//         check_eq(bob->receive(), "Hello Bob!");
 //         bob->trigger_handover = true;
 //         alice->send("Hello Carl!");
 //         alice->register_writing();
 //         bob->register_reading();
 //         exhaust();
-//         CHECK_EQ(bob->receive(), "");
-//         CHECK_EQ(bob->handle(), invalid_socket);
-//         if (CHECK_NE(bob->next, nullptr)) {
+//         check_eq(bob->receive(), "");
+//         check_eq(bob->handle(), invalid_socket);
+//         if (check_ne(bob->next, nullptr)) {
 //           auto carl = bob->next;
-//           CHECK_EQ(carl->handle(), socket{bob_fd});
+//           check_eq(carl->handle(), socket{bob_fd});
 //           carl->register_reading();
 //           exhaust();
-//           CHECK_EQ(carl->name, "Carl");
-//           CHECK_EQ(carl->receive(), "Hello Carl!");
+//           check_eq(carl->name, "Carl");
+//           check_eq(carl->receive(), "Hello Carl!");
 //         }
 //       }
 //     }
 //   }
 // }
```

Comment:
This is a commented out test which can be removed from the new tests. Are we planning to keep this test?

Reply:
It has a todo comment on it, so please leave it for now.